tgui/packages/tgui/interfaces/RapidConstructionDevice.tsx

```diff
@@ -0,0 +1,179 @@
+import { Window } from '../layouts';
+import { classes } from 'common/react';
+import { capitalizeAll } from 'common/string';
+import { useBackend, useLocalState } from '../backend';
+import { LabeledList, Section, Button, Tabs, Stack, Box } from '../components';
+import { AirLockMainSection } from './AirlockElectronics';
+
+type Data = {
+  matterLeft: Number;
+  space: Boolean;
+  silo_upgraded: Boolean;
+  silo_enabled: Boolean;
```

Comment:
```suggestion
  matterLeft: number;
  space: BooleanLike;
  silo_upgraded: BooleanLike;
  silo_enabled: BooleanLike;
```
same for the other instances